odata-protocol/11.4 Data Modification.md

```diff
@@ -341,8 +343,8 @@ request body. Missing non-key, updatable structural properties not
 defined as dependent properties within a referential constraint MUST be
 set to their default values. Omitting a non-nullable property with no
 service-generated or default value from a `PUT` request results in a
-`400 Bad Request` error. Missing dynamic structural properties MUST be
-removed or set to `null`.
+`400 Bad Request` error. Missing dynamic properties MUST be
```

Comment:
Missing = Omitted?

Reply:
Changed "missing" to "omitted" in this section